doc/users/explain/figures.rst

```diff
@@ -0,0 +1,209 @@
+
+Finally, the user can specify a layout engine for the figure with the *layout*
+parameter.  Currently Matplotlib supplies
+:doc:`"constrained" </tutorials/intermediate/constrainedlayout_guide>`,
+:ref:`"compressed" <compressed_layout>` and
+:doc:`"tight" </tutorials/intermediate/tight_layout_guide>` layout engines.  These
+rescale axes inside the Figure to prevent overlap of ticklabels, and try and align
+axes, and can save significant manual adjustment of artists on a Figure for many
+common cases.
+
+Adding Artists
+--------------
+
+The `~.FigureBase` class has a number of methods to add artists to a `~.Figure` or
+a `~.SubFigure`.  By far the most common are to add Axes of various configurations
+(`~.FigureBase.add_axes`, `~.FigureBase.add_subplot`, `~.FigureBase.subplots`,
+`~.FigureBase.subplot_mosaic`) and subfigures (`~.FigureBase.subfigures`).  Colorbars
+are added to Axes or group of Axes at the Figure level (`~.FigureBase.colorbar`).
+It is also possible to have a Figure-level legend (`~.FigureBase.legend`).
+Other Artists include figure-wide labels (`~.suptitle`,
+`~.subxlabel`, `~.subpylabel`) and text (`~.FigureBase.text`).  Finally, low-level
+Artists can be added directly using `~.FigureBase.add_artist` usually with care being
+taken to use the appropriate transform.  Usually these include ``Figure.transFigure``
+which ranges from 0 to 1 in each direction, and represents the fraction of the current
+Figure size, or ``Figure.dpi_scale_trans`` which will be in physical units of inches
+from the bottom left corner of the Figure (see
+:doc:`/tutorials/advanced/transforms_tutorial` for more details).
+
+
+.. _saving_figures:
+
+Saving Figures
+==============
+
+Finally, Figures can be saved to disk using the `~.Figure.savefig` method.
+``fig.savefig('MyFigure.png', dpi=200)`` will save a PNG formatted figure to
+the file ``MyFigure.png`` in the current directory on disk with 200 dots-per-inch
+resolution.  Note that the filename can include a relative or absolute path to
+any place on the file system.
+
+Many types of output are supported, including raster formats like PNG, GIF, JPEG,
+TIFF and vector formats like PDF, EPS, and SVG.
+
+By default, the size of the saved Figure is set by the Figure size (in inches) and, for the raster
+formats, the *dpi*.  If *dpi* is not set, then the *dpi* of the Figure is used.
+Note that *dpi* still has meaning for vector formats like PDF if the Figure includes
+Artists that have been :doc:`rasterized </gallery/misc/rasterization_demo>`; the
+*dpi* specified will be the resolution of the rasterized objects.
+
+It is possible to change the size of the Figure using the *bbox_inches* argument
+to savefig.  This can be specified manually, again in inches.  However, by far
+the most common use is ``bbox_inches='tight'``.  This option trims, or expands, the
+size of the figure so that it is tight around all the artists in a figure, with a
```

Comment:
Super picky word choice preference, feel free to disregard:

```suggestion
the most common use is ``bbox_inches='tight'``.  This option "shrink wraps,
trimming or expanding as needed, the size of the figure so that it is tight 
around all the artists in a figure, with a
```